src/commands/app/db/collection/insertOne.js

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { DBBaseCommand } from '../../../../DBBaseCommand.js'
+import { Args, Flags } from '@oclif/core'
+import chalk from 'chalk'
+
+export class InsertOne extends DBBaseCommand {
+  async run () {
+    const { collection, document } = this.args
+    const { bypassDocumentValidation } = this.flags
+
+    try {
+      // Parse document JSON
+      let documentObj
+      try {
+        documentObj = JSON.parse(document)
+      } catch (error) {
+        this.error(`Invalid document JSON: ${error.message}`)
+      }
+
+      this.log(chalk.blue(`Inserting document into collection '${collection}'...`))
+
+      if (bypassDocumentValidation) {
+        this.log(chalk.dim('   Bypassing document validation'))
+      }
+
+      const client = await this.db.connect()
+      const coll = client.collection(collection)
+
+      // Build options
+      const options = {}
+      if (bypassDocumentValidation) {
+        options.bypassDocumentValidation = true
+      }
+
+      // Insert the document
+      const result = await coll.insertOne(documentObj, options)
+
+      this.debugLogger?.info?.('Document inserted successfully:', result)
+
+      const response = {
+        collection,
+        document: documentObj,
+        insertedId: result.insertedId,
+        acknowledged: result.acknowledged,
+        namespace: this.rtNamespace,
+        timestamp: new Date().toISOString(),
+        result
+      }
+
+      this.log(chalk.green(`Document inserted successfully into collection '${collection}'`))
+      this.log(chalk.dim(`   Namespace: ${this.rtNamespace}`))
+      this.log(chalk.dim(`   Inserted ID: ${result.insertedId}`))
+      this.log(chalk.dim(`   Acknowledged: ${result.acknowledged}`))
```

Comment:
Unnecessary log, if acknowledged is false the operation wasn't performed and it should be an error case